wpilibj/src/athena/cpp/lib/HALUtil.cpp

```diff
@@ -258,6 +271,12 @@ JNIEXPORT jint JNICALL JNI_OnLoad(JavaVM *vm, void *reserved) {
   boundaryExCls = static_cast<jclass>(env->NewGlobalRef(local));
   if (!boundaryExCls) return JNI_ERR;
   env->DeleteLocalRef(local);
+  
+  local = env->FindClass("edu/wpi/first/wpilibj/util/AllocationException");
+  if (!local) return JNI_ERR;
+  allocationExCls = static_cast<jclass>(env->NewGlobalRef(local));
+  if (!boundaryExCls) return JNI_ERR;
```

Comment:
Should be allocationExCls.  I'll write a function at some point to refactor these.